seed/tasks.py

```diff
@@ -811,6 +826,8 @@ def _save_raw_data(file_pk, *args, **kwargs):
 
         if import_file.source_type == "Green Button Raw":
             return _save_raw_green_button_data(file_pk, *args, **kwargs)
+        #if import_file.source_type == "PM energy Raw":
```

Comment:
Hi Nicholas,

That function can be deleted, also the commented out part.

The uploaded energy template file is processed after save_raw_data()
function finished.

Haopeng

On Tue, Feb 9, 2016 at 11:02 AM, Nicholas Long <notifications@github.com>
wrote:

> In seed/tasks.py
> <https://github.com/SEED-platform/seed/pull/714#discussion_r52328124>:
>
> > @@ -811,6 +826,8 @@ def _save_raw_data(file_pk, *args, **kwargs):
> >
> >          if import_file.source_type == "Green Button Raw":
> >              return _save_raw_green_button_data(file_pk, *args, **kwargs)
> > +        #if import_file.source_type == "PM energy Raw":
>
> remove chunk
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/SEED-platform/seed/pull/714/files#r52328124>.
>